terraform/context_components_test.go

```diff
@@ -62,7 +62,7 @@ func simpleTestSchema() *configschema.Block {
 				Optional: true,
 			},
 			"test_number": {
-				Type:     cty.String,
+				Type:     cty.Number,
```

Comment:
Huh! Should we fix the other types in this schema below while we're at it?

Reply:
Since you've commented, I'll do it! It bothered me but I didn't want to expand scope too widely 😂